org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java

```diff
@@ -335,20 +338,40 @@ private Panel newGadgetPanel(MDashboardPreference dp, MDashboardContent dc) {
 		panel.setMaximizable(dc.isMaximizable());
 
 		String description = dc.get_Translation(MDashboardContent.COLUMNNAME_Description);
-	if(description != null)
-		panel.setTooltiptext(description);
-
-	panel.setCollapsible(dc.isCollapsible());
-	panel.setOpen(!dp.isCollapsedByDefault());
-	panel.addEventListener(Events.ON_OPEN, this);
-	if (!ClientInfo.isMobile()) {
-		panel.setDroppable("true");
-		panel.getCaption().setDraggable("true");
-		panel.addEventListener(Events.ON_DROP, this);
+		if(description != null) {
+			renderHelpButton(caption, description);
+		}
+		
+		panel.setCollapsible(dc.isCollapsible());
+		panel.setOpen(!dp.isCollapsedByDefault());
+		panel.addEventListener(Events.ON_OPEN, this);
+		if (!ClientInfo.isMobile()) {
+			panel.setDroppable("true");
+			panel.getCaption().setDraggable("true");
+			panel.addEventListener(Events.ON_DROP, this);
+		}
+		panel.setBorder("normal");
+	
+			return panel;
 	}
-	panel.setBorder("normal");
-
-		return panel;
+	
+	private void renderHelpButton(Caption caption, String text) {
+		ToolBarButton help = new ToolBarButton();
+		caption.appendChild(help);
+		if (ThemeManager.isUseFontIconForImage())
+			help.setIconSclass("z-icon-Help");
+		else
+			help.setImage(ThemeManager.getThemeResource(IMAGES_CONTEXT_HELP_PNG));
+		Popup popup = new Popup();
+		popup.setSclass("dashboard-content-help");
+		popup.setPopup(popup);
+		Text t = new Text(text);
+		popup.appendChild(t);
+		help.setTooltip(popup);
+		help.addEventListener(Events.ON_CLICK, (Event event) -> {
+			popup.setPage(help.getPage());
+			popup.open(help, "after_start");
```

Comment:
need to add LayoutUtils.autoDetachOnClose(popup) call. without which, can lead to memory leak.

Reply:
Didn't know that, thank you